common/notifications.cpp

```diff
@@ -733,25 +626,28 @@ namespace rs2
 
                 if (ImGui::Button("OK", ImVec2(120, 0)))
                 {
-                    selected.message = "";
+                    selected->message = "";
+                    selected = nullptr;
                     ImGui::CloseCurrentPopup();
                 }
-
-                std::string clip = "";
-                auto lines = split_string(selected.message, '\n');
-                for (auto line : lines)
-                {
-                    if (line.size() && line[0] == '$') clip += line.substr(2) + "\n";
-                }
-                if (clip != "")
+                else
                 {
-                    ImGui::SameLine();
-                    if (ImGui::Button(" Copy Commands "))
+                    std::string clip = "";
+                    auto lines = split_string(selected->message, '\n');
+                    for (auto line : lines)
                     {
-                        glfwSetClipboardString(win, clip.c_str());
+                        if (line.size() && line[0] == '$') clip += line.substr(2) + "\n";
+                    }
+                    if (clip != "")
+                    {
+                        ImGui::SameLine();
+                        if (ImGui::Button(" Copy Commands "))
+                        {
+                            glfwSetClipboardString(win, clip.c_str());
+                        }
+                        if (ImGui::IsItemActive())
+                            ImGui::SetTooltip("Paste the copied commands to a terminal and enter your password to run");
```

Comment:
What is this ?

Reply:
Copies command to clipboard. Since such commands would often start with `sudo`, the user will be prompted his password after pasting to terminal